src/iop/filmicrgb.c

```diff
@@ -196,62 +247,64 @@ int default_colorspace(dt_iop_module_t *self, dt_dev_pixelpipe_t *pipe, dt_dev_p
   return iop_cs_rgb;
 }
 
-void init_presets(dt_iop_module_so_t *self)
```

Comment:
You have removed all presets? Is that expected?

Reply:
Yes, I'm really not sure it's a good idea to have presets in there. People expect them to be one-size-fits-all so it's only frustration.

Reply:
Your take, but I use them as base work. I think people are used to them now.

Reply:
There is a large dependency to camera, for presets accuracy/efficiency, so I would rather encourage users to build their own.